Photo-Editor/Photo-Editor/Views/HomeScreenView.swift

```diff
@@ -26,12 +25,28 @@ struct HomeScreenView: View {
                     .ignoresSafeArea()
 
                 VStack {
+                    HStack {
+                        Text("Choose an image")
```

Comment:
This text field looks like a button. I understand that you are trying to make this stand out, but it could confuse the user.  This pattern appears an multiple views.